backend/app/api/endpoints/adapter/models.py

```
@@ -266,6 +266,138 @@ def bulk_create_models(
+    }
+    """
+    from app.schemas.kind import Model as ModelCRD
+
+    # Query all active Model CRDs from kinds table
+    models = (
+        db.query(Kind)
+        .filter(
+            Kind.user_id == current_user.id,
+            Kind.kind == "Model",
+            Kind.namespace == "default",
+            Kind.is_active == True,
```

Ruff (0.15.9)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)